pkg/server/server.go

```diff
@@ -183,6 +183,10 @@ func (s *Server) Run(ctx context.Context) error {
 		return err
 	}
 
+	if err := s.options.GenericControlPlane.Audit.ApplyTo(genericConfig); err != nil {
+		return err
+	}
+
```

Comment:
oh, is this really needed? Don't we already call it through the options package somehow?

Reply:
If we don't and you check this lgtm.

Reply:
I did a local test where I did: 

```
go run ./cmd/kcp start --audit-log-maxage=10 --audit-log-maxbackup=1 --audit-log-maxsize=100 --audit-log-path=audit.log --audit-policy-file=audit-policy.yml
```

and no `audit.log` was created.  I then added the code in this PR, ran the same command, and the `audit.log` was created.